<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBanner</class>
 <widget class="QWidget" name="SearchBanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>26</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="searchInfo"/>
   </item>
   <item>
    <widget class="QToolButton" name="highlightAllButton">
     <property name="visible">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Highlight All</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="previousButton">
     <property name="styleSheet">
      <string notr="true">border: none;
background: lightgrey;
padding: 0 5;
height: 30px;
border-radius: 2px;
</string>
     </property>
     <property name="text">
      <string notr="true">&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="nextButton">
     <property name="styleSheet">
      <string notr="true">border: none;
background: lightgrey;
padding: 0 5;
height: 30px;
border-radius: 2px;
</string>
     </property>
     <property name="text">
      <string notr="true">&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="SearchLineEdit" name="searchLineEdit">
     <property name="styleSheet">
      <string notr="true">border: none;
height: 30px;
border-radius: 2px;
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="doneButton">
     <property name="styleSheet">
      <string notr="true">border: none;
background: lightgrey;
padding: 0 20;
height: 30px;
border-radius: 2px;
text-align: center;

</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>searchlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
